marklogic/README.md

```diff
@@ -0,0 +1,96 @@
+# Agent Check: MarkLogic
+
+## Overview
+
+This check monitors [MarkLogic][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The MarkLogic check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+#### Prepare MarkLogic
+
+Using the API or the Admin interface, create a user for the Datadog Agent with at least the [`manage-admin`][3] role.
```

Comment:
`manage-user` role is not enough if we want to access the `/manage/v2?view=health` endpoint.

Reply:
That's expected to work with `datadog:datadog` because it's already a `manage-admin` (see https://github.com/DataDog/integrations-core/pull/6524/files#diff-eb070e813473dda6f8d12fc351573c38R67)

If you spin up the e2e environment create a `manage-user` and try to access this endpoint, you should get an error.
```
curl -X POST --anyauth --user admin:admin -i -H "Content-Type: application/json" -d '{"user-name": "datadog2", "password": "datadog", "roles": {"role": "manage-user"}}' http://localhost:8002/manage/v2/users
curl -X GET --anyauth --user datadog2:datadog -i http://localhost:8002/manage/v2\?view\=health\&format\=json
```

Reply:
The health service checks are now behind a configuration option `enable_health_service_checks` disable by default.
When enabled, it needs the `manage-admin` permission, otherwise `manage-user` is enough.